WordPress/WordPressShareExtension/ShareModularViewController.swift

```diff
@@ -445,17 +463,26 @@ fileprivate extension ShareModularViewController {
 
     func isModulesSectionEmpty(_ sectionIndex: Int) -> Bool {
         switch ModulesSection(rawValue: sectionIndex)! {
-        case .categories:
+        case .type:
             return false
+        case .categories:
+            return shareData.postType == .page
         case .tags:
-            return false
+            return shareData.postType == .page
```

Comment:
Yes, I should think so, but I don't see way to add tags when creating a page in the app. When creating a post in-app, you get a prompt to add categories and tags, but not for posts - unless I'm overlooking something? I thought it best to keep consistency with the app experience.

Reply:
@osullivanchris - Can I ask your thoughts on this?

To summarize:

Should the share extension allow users to add tags to pages, even though the App itself doesn't let you edit the tags for a post?  The web page does let you add tags to a page.

My personal opinion is we can enable it in the share extension, and eventually add it in the Page settings in the App later on.

Reply:
Sending up Tags in a Page creation request doesn't _just work_, so this might be harder than I imagined. Not to doubt you, @diegoreymendez (😅) but are you certain you can add Tags to a Page? I don't see the option when creating a page, myself, and the [glossary seems to suggest they don't](https://wordpress.org/support/article/post-types/#pages).

Reply:
> are you certain you can add Tags to a Page?

Not at all!  I just know it's possible to add them in the web editor, but I initially posed this as a question in case you found anything that made this undesirable, complicated, or straight out impossible.

Considering your findings I'd say it's ok to just resolve this and go forward without tags.

If necessary, it can be evaluated and implemented separately in the future.

Reply:
> My personal opinion is we can enable it in the share extension, and eventually add it in the Page settings in the App later on.

that seems fine to me.

> but are you certain you can add Tags to a Page?

from a very brief exploration it seems possible for me on WordPress.com sites. For a dotorg site it is not there by default, but I think it can be enabled with a plugin.